### PR TITLE
Update Lumen Supply Metrics

### PR DESCRIPTION
**Pull request by briwylde08**

### CODE REVIEW

File: docs/learn/encyclopedia/lumen-supply-metrics.mdx [hunk 1]
```diff
@@ -39,7 +41,7 @@ As of December 12, 2019, the Dashboard API shows:
 
 **inflationLumens** For the first five or so years of Stellar’s existence, the supply of lumens increased by 1% annually. This “network inflation” was ended by validator vote on October 28, 2019. The total number of lumens generated by inflation was 5,443,902,087.3472865.
 
-Adding this number to the Original Supply, you get the total lumens that have ever existed: 105,443,902,087.3472865. This number is visible on the List All Ledgers (`https://horizon-mainnet-url/ledgers?order=desc`) Horizon API endpoint as \_embedded.records.total_coins.
+Adding this number to the Original Supply, you get the total lumens that have ever existed: 105,443,902,087.3472865. This number is visible on the List All Ledgers (`https://horizon.stellar.org/ledgers?order=desc`) Horizon API endpoint as \_embedded.records.total_coins.
```
Comment (janewang) [inline docs/learn/encyclopedia/lumen-supply-metrics.mdx:44]: We could link out to pages in our docs and to the Horizon community providers:

The number is visible on the [List All Ledgers](https://developers.stellar.org/network/horizon/api-reference/resources/list-all-ledgers) Horizon API as \_embedded.records.total_coins when called from a mainnet Horizon(https://developers.stellar.org/network/horizon/horizon-providers).

File: docs/learn/encyclopedia/lumen-supply-metrics.mdx [hunk 2]
```diff
@@ -49,7 +51,7 @@ While any address with no signers is counted here, the vast majority of the lume
...
+**feePool** The Fee Pool is where network fees collect. The lumens do not belong to any particular account. No one has access to the fee pool, so these lumens are non-circulating. Network validators could theoretically vote for a protocol change that would affect the fee pool, so we include it n the total supply. Stellar’s transaction fees are extremely low so the fee pool grows very slowly. The Fee Pool is tracked by the protocol itself, and the current number is visible on the List All Ledgers (`https://horizon.stellar.org/ledgers?order=desc`) Horizon API endpoint as `_embedded.records.fee_pool`. i **sdfMandate** The SDF Mandate is described in detail [here](https://www.stellar.org/foundation/mandate). The Foundation was funded by lumens generated at Stellar’s inception; all of those lumens will eventually be spent or distributed to enhance and promote Stellar. Here is a complete list of the addresses currently associated with the SDF Mandate:
```
Comment (janewang) [inline docs/learn/encyclopedia/lumen-supply-metrics.mdx:54]: Similar here to link to Ledger Horizon API page and Horizon community providers.

**Comment by briwylde08** [comment]
Thanks @janewang lmk what you think of the edits

**Comment by janewang** [comment]
Thank you Bri

**Review by janewang — APPROVED** [review]

**Pull request merged**